components/dashboard/src/components/PendingChangesDropdown.tsx

```diff
@@ -4,12 +4,49 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { WorkspaceInstance } from "@gitpod/gitpod-protocol";
+import { WorkspaceInstance, WorkspaceInstanceRepoStatus } from "@gitpod/gitpod-protocol";
 import ContextMenu, { ContextMenuEntry } from "./ContextMenu";
 import CaretDown from "../icons/CaretDown.svg";
+import { useQuery } from "@tanstack/react-query";
+
+const useGitStatusQuery = ({ workspaceInstance }: { workspaceInstance?: WorkspaceInstance }) => {
+    return useQuery<WorkspaceInstanceRepoStatus>({
+        queryKey: [
+            "git-status",
+            workspaceInstance?.id ?? "",
+            workspaceInstance?.status.phase ?? "",
+            workspaceInstance?.ideUrl ?? "",
+        ],
+        queryFn: async () => {
+            if (!workspaceInstance) {
+                return undefined;
+            }
+            if (workspaceInstance.status.phase !== "running") {
+                return workspaceInstance.status.repo;
+            }
+            const workspaceUrl = new URL(workspaceInstance.ideUrl);
+            workspaceUrl.pathname = "/_supervisor/v1/status/git";
+            try {
+                const response = await fetch(workspaceUrl.toString(), { credentials: "include" });
```

Comment:
maybe need owner token for auth

Reply:
most likely owner cookie is already installed via the workspace loaciotn already, let's try like that